src/nwb/NWBFile.cpp

```diff
@@ -41,44 +42,44 @@ NWBFile::~NWBFile() {}
 Status NWBFile::initialize(const std::string description,
                            const std::string dataCollection)
 {
-  if (std::filesystem::exists(io->getFileName())) {
-    return io->open(false);
+  if (std::filesystem::exists(this->m_io->getFileName())) {
+    return this->m_io->open(false);
   } else {
-    io->open(true);
+    this->m_io->open(true);
     return createFileStructure(description, dataCollection);
   }
 }
 
 Status NWBFile::finalize()
 {
-  return io->close();
+  return this->m_io->close();
 }
 
 Status NWBFile::createFileStructure(std::string description,
                                     std::string dataCollection)
 {
-  if (!io->canModifyObjects()) {
+  if (!this->m_io->canModifyObjects()) {
     return Status::Failure;
   }
 
-  io->createCommonNWBAttributes("/", "core", "NWBFile", "");
-  io->createAttribute(AQNWB::SPEC::CORE::version, "/", "nwb_version");
-
-  io->createGroup("/acquisition");
-  io->createGroup("/analysis");
-  io->createGroup("/processing");
-  io->createGroup("/stimulus");
-  io->createGroup("/stimulus/presentation");
-  io->createGroup("/stimulus/templates");
-  io->createGroup("/general");
-  io->createGroup("/general/devices");
-  io->createGroup("/general/extracellular_ephys");
+  this->m_io->createCommonNWBAttributes("/", "core", "NWBFile", "");
```

Comment:
several cases using both `this->` and `m_io` in this file that I think could be simplified to `m_io`